ngx-fudis/projects/ngx-fudis/src/lib/components/form/guidance/guidance.component.html

```diff
@@ -85,13 +85,15 @@
       <ng-container *ngIf="helpText">{{ helpText }}</ng-container>
       <ng-container *ngIf="control && maxLength">
         <span class="fudis-visually-hidden"
-          >{{ control.value?.length || 0 }}/{{ maxLength }} {{ _maxLengthText | async }}</span
+          >&nbsp; {{ control.value?.length || 0 }}/{{ maxLength }}
+          {{ _maxLengthText | async }}</span
         >
       </ng-container>
     </p>
   </div>
   <ng-container *ngIf="maxLength && control">
     <small
+      aria-hidden="true"
```

Comment:
Remove this aria-hidden and add visually hidden text after the maxLength indicator, like so:

```
      {{ control.value?.length || 0 }}/{{ maxLength
      }}<span class="fudis-visually-hidden">{{ _maxLengthText | async }}</span>
```